### PR TITLE
Add cd to dockerhub

### PR DESCRIPTION
**Pull request by Quarzerback**

### CODE REVIEW

File: .github/workflows/deploy-to-render.yml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+          password: ${{ secrets.DOCKERHUB_PASSWORD }} # must match the password of the Dockerhub account
+
+      - name: Build and push
+        uses: docker/build-push-action@v5
+        with:
+          push: true
+          tags: ${{ secrets.DOCKERHUB_TAG }} # Example: username/project:latest
+          context: .
+
+  deploy:
```
Comment (Khorm247) [inline .github/workflows/deploy-to-render.yml:79]: are you sure you already need a deploy part? :)

did the same mistake, on my project ;)

**Review by Khorm247 — CHANGES_REQUESTED** [review]
name: Capstone Project # Capstone Project name
url: https://neuefische.de/ # Link to deployment

File: .github/workflows/deploy-to-render.yml (hunk 1 above)
Comment (Quarzerback) [reply .github/workflows/deploy-to-render.yml]: thanks, removed

**Review by Khorm247 — COMMENTED** [review]
lgtm

**Review by Khorm247 — APPROVED** [review]

**Review by Khorm247 — APPROVED** [review]

**Review by Khorm247 — APPROVED** [review]
lgtm

**Review by uurkrtl — APPROVED** [review]
LGTM

**Pull request merged**